core/native_type_traits.h

```diff
@@ -55,6 +77,60 @@ struct NativeTypeTraits<IDLDouble> : public NativeTypeTraitsBase<IDLDouble> {
   }          
 };
 
+/*
+ * The float type is a floating point numeric type that corresponds to the set
+ * of finite single-precision 32 bit IEEE 754 floating point numbers.
+ */
+
+template <>
+struct NativeTypeTraits<IDLFloat> : public NativeTypeTraitsBase<IDLFloat> {
+  static float NativeValue(const Napi::Env& env, const Napi::Value& js_value) {
+    if (!js_value.IsNumber()) {
+      Napi::TypeError::New(env, "It's an invalid number.")
+          .ThrowAsJavaScriptException();
+      return 0.0;
```

Comment:
Shoud be 0.0f

Reply:
I removed float type, I will update float type soon.